src/azure-cli/azure/cli/command_modules/synapse/commands.py

```diff
@@ -200,13 +211,35 @@ def get_custom_sdk(custom_module, client_factory):
         g.show_command('show', 'get')
         g.generic_update_command('update', custom_func_name='sqlpool_blob_auditing_policy_update')
 
+    # Management Plane Commands --Sql Ad-Admin
+    with self.command_group('synapse sql ad-admin', command_type=synapse_workspace_aad_admin_sdk,
+                            custom_command_type=get_custom_sdk('workspacesqlaadadmin',
+                                                               cf_synapse_client_workspace_aad_admins_factory),
+                            client_factory=cf_synapse_client_workspace_aad_admins_factory) as g:
+        g.show_command('show', 'get')
+        g.custom_command('create', 'create_workspace_sql_aad_admin', supports_no_wait=True)
+        g.generic_update_command('update', custom_func_name='update_workspace_sql_aad_admin',
+                                 setter_arg_name='aad_admin_info', supports_no_wait=True)
+        g.command('delete', 'delete', confirmation=True, supports_no_wait=True)
+        g.wait_command('wait')
+
+    # Management Plane Commands --Sql audit-policy
+    with self.command_group('synapse sql audit-policy',
+                            command_type=synapse_workspace_managed_sqlserver_blob_auditing_policies_sdk,
+                            custom_command_type=get_custom_sdk('sqlpoolblobauditingpolicy',
+                                                               cf_synapse_client_sqlserver_blob_auditing_policies_factory),
+                            client_factory=cf_synapse_client_sqlserver_blob_auditing_policies_factory) as g:
+        g.show_command('show', 'get')
+        g.generic_update_command('update', custom_func_name='sqlserver_blob_auditing_policy_update')
```

Comment:
only two commands for audit-policy? What about delete, create, list?

Reply:
Here we just to keep consistent with `az sql server`. And I checked for `az synapse sql audit-policy` the dependent api is a long running, and for `az synapse sql pool audit-policy` the dependent api is not long running.
I have add wait command and add `suppport_no_wait=True` for this comments.